addons/l10n_ug/data/template/account.account-ug.csv

```diff
@@ -0,0 +1,259 @@
+310425,"Entertainment, Literary and Artistic Originals",310425,asset_fixed,,False
+310431,Copy Rights and Neighboring Rights,310431,asset_fixed,,False
+310432,Patents / Utility Modes,310432,asset_fixed,,False
+310433,Trade Secrets,310433,asset_fixed,,False
+310441,Goodwill and marketing assets,310441,asset_fixed,,False
+310449,Other Intellectual Property products,310449,asset_fixed,,False
+320111,Materials and supplies,320111,asset_prepayments,,False
+320112,Work in progress,320112,asset_prepayments,,False
+320113,Finished goods,320113,asset_prepayments,,False
+320114,Goods for resale,320114,asset_prepayments,,False
+320115,Military Inventories,320115,asset_prepayments,,False
+320119,Other Inventories,320119,asset_prepayments,,False
+330111,National treasures,330111,asset_prepayments,,False
+340111,Land,340111,asset_fixed,,False
+340211,Minerals,340211,asset_fixed,,False
+340212,Oil & Natural Gas,340212,asset_fixed,,False
+340213,Energy resources,340213,asset_fixed,,False
+340219,Other Mineral and Energy Resources,340219,asset_fixed,,False
+340311,Wild Animals,340311,asset_fixed,,False
+340312,Plants,340312,asset_fixed,,False
+340321,Lakes,340321,asset_fixed,,False
+340322,Rivers and Streams,340322,asset_fixed,,False
+340323,Swamps and Wetlands,340323,asset_fixed,,False
+340329,Other Water resources,340329,asset_fixed,,False
+340331,Radio Spectrum - Other Natural Resources,340331,asset_fixed,,False
+340332,Aviation Airspace,340332,asset_fixed,,False
+340341,Mountains and Rocks,340341,asset_fixed,,False
+340349,Other Natural Resources (Not classified elsewhere),340349,asset_fixed,,False
+340401,"Marketable operating leases - Contracts, leases and Permits",340401,asset_non_current,,False
+340402,"Permits to use natural resources - Contracts, leases and Permits",340402,asset_non_current,,False
+340403,"Permits to undertake specific activities - Contracts, leases and Permits",340403,asset_non_current,,False
+340405,Entitlement to future goods and services on an exclusive basis - Contracts leases and Permits,340405,asset_non_current,,False
+340406,Goodwill and marketing assets,340406,asset_non_current,,False
+3510,Monetary Gold and SDRs,3510,asset_current,,False
+3522,Debt Securities,3522,asset_current,,False
+3523,Loans,3523,asset_current,,False
+352401,Shares in public corporations,352401,asset_non_current,,False
+352402,Shares in private entities,352402,asset_non_current,,False
+352501,Petroleum Revenue Investment Reserve,352501,asset_non_current,,False
+352599,Other Investment Fund Shares or Units,352599,asset_non_current,,False
+352701,Forwards,352701,asset_prepayments,,False
+352702,Futures,352702,asset_prepayments,,False
+352703,Options,352703,asset_prepayments,,False
+352704,Swaps,352704,asset_prepayments,,False
+3528,Account Receivable,3528,asset_receivable,,True
+352802,Staff Advances,352802,asset_receivable,,True
+352803,Other Advances,352803,asset_receivable,,True
+352804,Taxes Receivable,352804,asset_receivable,,True
```

Comment:
Needs to be current_assets.  Normally we do not have that many receivable accounts.